deps.windows/60-qrcodegencpp.ps1

```diff
@@ -108,4 +108,17 @@ function Fixup {
             Remove-Item -Recurse -Force $Item.FullPath
         }
     }
+    $Items = @(
```

Comment:
As mentioned off-thread, this should either be handled by setting `CMAKE_DEBUG_POSTFIX` (to enforce the `d` suffix) or if the file is renamed manually, the generated `qrcodegencpp-targets-debug.cmake` file needs to be patched with the new filename (as otherwise CMake will pick up the non-debug variant for debug again).

Reply:
I've addressed this in a force push to use `CMAKE_DEBUG_POSTFIX`, which is also what I had done locally and did not have time to push up anywhere.